src/OrbitGl/UIElement.h

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_UI_ELEMENT_H_
+#define ORBIT_GL_UI_ELEMENT_H_
+
+#include "PickingManager.h"
+
+class TimeGraph;
+class GlCanvas;
+
+namespace orbit_gl {
+
+/* Base class for UI elements drawn underneath the capture window. */
+class UIElement : public Pickable {
```

Comment:
I would really prefer a more specific name that hints to the fact that this is an element of the capture window, as the comment says. For example, `CaptureWindowElement` or `CaptureViewElement` or `TimeGraphElement` something like that (not sure what our favorite way of calling the capture view is).

Reply:
Agreed, I changed it to `CaptureViewElement`.